lambdas/jwtverification/src/main/java/uk/gov/di/ipv/cri/passport/jwtverification/JwtVerificationHandler.java

```diff
@@ -17,23 +23,50 @@
 public class JwtVerificationHandler
         implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
 
+    private final ConfigurationService configurationService;
+
     private static final Logger LOGGER = Logger.getLogger(JwtVerificationHandler.class.getName());
     private static final Integer OK = 200;
     private static final Integer BAD_REQUEST = 400;
+    private static final String CLIENT_ID = "client_id";
+
+    public JwtVerificationHandler(ConfigurationService configurationService) {
+        this.configurationService = configurationService;
+    }
+
+    @ExcludeFromGeneratedCoverageReport
+    public JwtVerificationHandler() {
+        this.configurationService = new ConfigurationService();
+    }
 
     @Override
     public APIGatewayProxyResponseEvent handleRequest(APIGatewayProxyRequestEvent input, Context context) {
+
+        String ClientId = RequestHelper.getHeaderByKey(input.getHeaders(), CLIENT_ID);
```

Comment:
This variable name should be camelCase, so lowercase 'c' to start with

Reply:
Refactored